packages/atoms/src/Icon/Icon.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {
+    Canvas,
+    ImageSVG,
+    useSVG,
+    Group,
+    Paint,
+    BlendColor,
+    usePaintRef,
+} from '@shopify/react-native-skia';
+import { IconType, iconTypes } from './iconTypes';
+import { useNativeStyles } from '@trezor/styles';
+import { Color } from '@trezor/theme';
+
+const iconSizes = {
+    small: 16,
+    big: 24,
+} as const;
+
+export type IconSize = keyof typeof iconSizes;
+
+export type IconProps = {
+    type: IconType;
+    size?: IconSize;
+    color?: Color;
+};
+
+export const Icon = ({ type, size = 'big', color = 'black' }: IconProps) => {
```

Comment:
Maybe also pass a `className` to be used with SC?

Reply:
Sadly Skia is available only in RN, Chrome uses it too, but for rendering under the hood, so we can't access it.

I am probably going to rename whole package to `atoms-native` since current state starting to be confusing.

Reply:
In web browser you can easily achieve same behavior using plain <img> and css `fill` property.

Reply:
If it's native only, does it still make sense to have it as a separate package?

Reply:
Purpose of packages it's not only reusability but also modularity and separation of concerns. It's whole different way of thinking and structuring projects in monorepo.

Monorepo is supposed to consinsts big number of small packages could even hundreds or thousands. It quite crucial because soon we are going to implement Nx or Turborepo because our CI runs for like 45 minutes now. These tools can run CI task only for affected packages so it will be significant boost. But for running correctly it needs small packages to run on smallest possible part of packages tree. That will never work with just folder structure.

I am planning to discuss some changes in global packages structure, long story short I plan to introduce multiple packages folders instead of just `packages/` there will be probably something like `packages-native/` and `packages-connect/` alongside of `packages/`. I plan to discuss this on next code improvements.